src/assets/scss/modules/_buttons.scss

```
@@ -157,6 +157,11 @@
       opacity: 0.4;
     }
 
+    &.dp-button-disabled {
+      pointer-events: none;
+      opacity: 0.4;
+    }
+
```

Did we not already have disabled buttons?

---

In the library we use the disabled element property, this class is for old doppler elements that don't use this property

---

Oh, I understand. And using _disabled element property_ in doppler is not possible?

If not, you could add a comment to make it explicit that it is a patch and not the most important usage, for example:

```suggestion
    // Prefer using disabled element property
    // The class dp-button-disabled should only be used when using disabled property is not an option
    &.dp-button-disabled {
      pointer-events: none;
      opacity: 0.4;
    }

```